srcpkgs/simutron/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'simutron'
+pkgname=simutron
+version=1.0.1
+revision=1
+_rev=SR2
+build_wrksrc=build
+build_style=qmake
+hostmakedepends="subversion qt5-qmake qt5-host-tools pkg-config"
+makedepends="simavr-devel qt5-devel"
+depends="libelf"
+short_desc="AVR simulator IDE"
+maintainer="Artur Sinila <opensource@logarithmus.dev>"
+license="GPL-3.0-only"
+homepage="https://sourceforge.net/projects/simutron/"
+nostrip_files="LCD20x4Test.elf"
+
+if [ "$CROSS_BUILD" ]; then
+	hostmakedepends+=" qt5-host-tools"
+fi
+
+do_fetch() {
+	svn checkout "https://svn.code.sf.net/p/${pkgname}/code/branches/RB-${version}-${_rev}" "$wrksrc"
+}
+
+do_extract() {
+	echo "No need to extract"
```

Comment:
`:`

Reply:
What do you mean by `:`?

Reply:
If distfiles is empty you don't need to define this function at all.

That said, can't svn.code.sf.net provide some sort of tarball? :/